makefile

```diff
@@ -23,23 +23,27 @@ filenamesc := ft_strlen.c \
 			ft_strncmp.c \
 			ft_atoi.c \
 
-filenameso := $(filenamesc:%.c=%.o) 
-CC = gcc -Wall -Wextra -Werror
+filenameso := $(filenamesc:%.c=%.o)
+
+CC = gcc
+CFLAGS = -Wall -Wextra -Werror
+NAME = libft.a
```

Comment:
I'm not sure I get the advantage of not hardcoding it, here. Doesn't adding a variable just make things harder for that poor little computer of ours?